src/browser/main.coffee

```diff
@@ -54,17 +54,20 @@ start = ->
       else
         path.resolve(pathToOpen)
 
-    if args.devMode
-      AtomApplication = require path.join(args.resourcePath, 'src', 'browser', 'atom-application')
-    else
-      AtomApplication = require './atom-application'
+    AtomApplication = require path.join(args.resourcePath, 'src', 'browser', 'atom-application')
```

Comment:
@kevinsawicki Could you :eyes: this part? My goal removing this conditional was to cut off the problematic drive name at the earliest point possible, which is basically that `__filename` and `__dirname` and friends can be incorrect in `main.js`. This problem get propagated to other files if we call `require` with a relative path.